doctr/io/pdf.py

```diff
@@ -28,8 +29,9 @@ def read_pdf(
     Args:
         file: the path to the PDF file
         scale: rendering scale (1 corresponds to 72dpi)
+        rgb_mode: if True, the output will be RGB, otherwise BGR
```

Comment:
Can doctr work with BGR input, too?

Reply:
working yes but it would not be correct :sweat_smile: but we need to keep it flexible for users which to anything else with the loaded file

Reply:
Example:
x = DocumentFile.from_pdf(..)
do with the numpy arrays what you want to do :)

Reply:
Yep, I see